Sources/Workspace/InitPackage.swift

```diff
@@ -213,6 +216,15 @@ public final class InitPackage {
                             targets: ["\(pkgname)"]),
                     ]
                 """)
+            } else if packageType == .buildToolPlugin || packageType == .commandPlugin {
+                pkgParams.append("""
+                    products: [
+                        // Products define the executables and libraries a package produces, making them visible to other packages.
```

Comment:
This comment seems odd here, since our product is neither.

Reply:
Good point.  I just went with the stock one, but will refine it a bit.

Reply:
Updated